### DISCUSSION TITLE
Yocto Project Implementation

### DISCUSSION THREAD

**Discussion by maitreyakale** [polarfire-soc/polarfire-soc-documentation#333] [General]
I am trying to build a Linux Image with a root file system with a main branch i.e. **MACHINE=icicle-kit-es bitbake core-image-minimal-dev** but following error logs are coming.
![core_image_minimal](https://github.com/polarfire-soc/polarfire-soc-documentation/assets/109587121/9dca5f23-54bf-41fc-a3cb-bd59a6029bdb)

I also tried with the RAM-based Root Filesystem (Initramfs)
[init_ramfs_error_logs.txt](https://github.com/polarfire-soc/polarfire-soc-documentation/files/11686979/init_ramfs_error_logs.txt)
but got errors,please find the attached txt file.

Please Provide Solution onto this.

Thank You

**Comment by rockoguido** [comment]
hello, I replicated your scenario with the latest branch.  2023.02.1 and I've seen this before when some sites are in a wierd state where various packages reside, the downloads fail and errors like yours are asserted.     Just run bitbake again and you should be ok.   This is the approach I did, just banged it into submission.  :)  2nd time around everything built.  
Sometimes Yocto seems to need a little adult supervision.  
Hope this helps.. 
Dave

**Reply by maitreyakale** [reply]
Thank You for your reply!
